RELEASE-NOTES.md

```diff
@@ -5,6 +5,194 @@ issues and limitations, and legal information. To learn more, refer to the
+
+*  Replace all uses of the term "experience kit" with "package/kit"
+
+*  Significantly reduce build and deployment time for EC cluster and service deployment
+
+*  Upgrade latest version of supported Cluster for v0.5.0 (Need a list)
+
+*  Upgrade supported services for V0.5.0 （Need a list)
+
+*  Enable Debian 11.3 OS support
+
+*  Enable Ubuntu 22.04 OS & preempt_RT support
+
+*  Add Cert-Management support
+
+*  Enable error index for the error message
+
+*  Add 3 tutorials of "Hello Cluster Helm", "Web-info" and "RKE deployment"
+
+## Feature & Version List for Cluster Type v0.5.0
+
+The following table lists available features and versions for each cluster type
+provided in the Edge Conductor v0.5.0 release.
+
+| No.| EC v0.5.0 Features            | Version | Kind | RKE  | CAPI |
+|----| -----------------------------|---------|------|------|----- |
+| 1  |`Calico`                 |  3.23.1    | No  | No | Yes   |
+| 2  |`CRI-O`                       |  1.23.2 | No   | No   | Yes  |
+| 3  |`Edge Software Provisioner`   |  2.0.3  | No   | Yes  | Yes† |
+| 4  |`Grafana Dashboard`           |  8.3.6  | Yes  | Yes  | Yes  |
+| 5  |`Intel-GPU-Plugin`            |  0.23   | No   | Yes  | No   |
+| 6  |`Multus`                      |  0.3.1   | Yes  | Yes  | Yes  |
+| 7  |`Offline deployment`          |  N/A    | No   | Yes  | Yes†   |
+| 8  |`Portainer-ce`                |  1.0.32 | Yes  | Yes  | Yes  |
+| 9  |`Prometheus`                  |  36.2.1 | Yes  | Yes  | Yes  |
+| 10 |`Rook-ceph`                   |  1.9.6  | No   | Yes  | Yes  |
+| 11 |`Rook-ceph-cluster`           |  1.9.6  | No   | Yes  | Yes  |
+| 12 |`Intel-sriov-network`                      |  3.5.1    | No   | Yes  | Yes  |
+| 13 |`Akri`                        |  0.8.4  | Yes  | Yes  | Yes  |
+| 14 |`NFD`                         |  0.11.0 | No   | Yes  | Yes  |
+| 15 |`rt-linux-detection`          |  N/A    | Yes  | Yes  | Yes  |
+| 16 |`nginix-ingress`          |  4.2.0    | No  | No  | Yes  |
+| 17 | `CPU Manager`                 |  N/A    | Yes  | Yes  | No   |
+| 18 |`kubevirt-operator`          |  0.54.0    | Yes  | Yes  | Yes  |
+| 19 |`kubevirt-cr`          |  0.54.1    | Yes  | Yes  | Yes  |
+
+†  Note that Edge Software Provisioner and offline deployment feature is enabled on CAPI clusters when using the BYOH deployment framework.
+
+The following table lists the versions for each cluster type provided in the Edge Conductor v0.5.0 release.
+| No.| Cluster            | Version | K8S version | CRI Version  | Other Version | Comments|
+|----| ------------|---------|---------|-----------|---------------|------------|
+| 1  |`KIND`               |  0.14.0    | 1.24.2  | -- | -- |-- |
+| 2  |`RKE`                 |  1.3.12    | 1.23.7-rancher1-1(default)  | -- | -- |-- |
+| 3  |`CAPI(core)`      |  1.23.2 | ---   | ---   | Cert-manager v1.9.0 |verified with metal3 v1.1.1|
+| 4  |`CAPI-BYOH`    |  1.1.2 | 1.23.5   |  Crio v1.23.3 Containerd v1.6.6 | -- |---|
```

Comment:
should be 
| 4  |`CAPI-BYOH`    |  0.2.0 | 1.23.5   |  Crio v1.23.3 Containerd v1.6.6 | -- |---|